src/vm/threadsuspend.cpp

```diff
@@ -6853,7 +6880,6 @@ void Thread::HijackThread(VOID *pvHijackAddr, ExecutionState *esb)
 
 #ifdef _DEBUG
     static int  EnterCount = 0;
-    _ASSERTE(EnterCount++ == 0);
```

Comment:
I removed that assert while debugging, but I'm going to have to change `EnterCount` to not be a static local variable. The reason is that now we can have multiple threads being hijacked at the same time. I'll address this with my next push.